storage.js

```diff
@@ -0,0 +1,38 @@
+// Retrieve the form data from local storage
+function getFormData() {
+  const formData = localStorage.getItem('formData');
+  return formData ? JSON.parse(formData) : {};
+}
+
+// Save the form data to local storage
+function saveFormData(data) {
+  localStorage.setItem('formData', JSON.stringify(data));
+}
+
+// Update the form fields with the saved data
+function updateFormFields() {
+  const formData = getFormData();
+  document.getElementById('fullname').value = formData.fullname || '';
+  document.getElementById('email').value = formData.email || '';
+  document.getElementById('message').value = formData.message || '';
+}
+
+// Save the form data whenever a field is changed
+function handleInputChange(event) {
+  const inputField = event.target;
+  const fieldName = inputField.fullname;
```

Comment:
kudos on reaching this milestone, this milestone focuses on `preserving data in the browser`  and it seems the requirement is not met. This is the expected behavior of the project 

- The contact form information is persistent to page reloads.

- After filling the form, and not submitting it, the entered values should be saved in the browser.

- When visiting the page the form inputs should be contained, be auto-populated with, the values entered in the previous visit to the page.

- If at the previous visit to the page, in the same browser, no input was entered in the form then the form inputs should remain empty.

This is the current behaviour of your project :


https://github.com/yayaismaail/my-portfolio/assets/20537712/1a3feee7-c697-4320-90f8-0fae57113adb

Reply:
@usorfaitheloho, please I need your assistance because I have tried all my best but I couldn't find the solution